mypy/checkexpr.py

```diff
@@ -2806,10 +2797,7 @@ def dangerous_comparison(
         return not is_overlapping_types(left, right, ignore_promotions=False)
 
     def get_operator_method(self, op: str) -> str:
```

Comment:
Maybe just delete this method, and replace its usages with direct dictionary lookups? Same with `get_reverse_op_method`